simd.dd

```diff
@@ -92,8 +100,8 @@ $(H3 Conversions)
 	to their vector equivalents:)
 
 ---
-int4 v = 7;
```

Comment:
Float multiplication works just fine. It's int multiplication that doesn't
tend to work for ints and longs. I'm tempted to just change it to adding
int4's instead of multiplying. It avoids the confusion and is the smallest
change from the existing code.

On Wed, Jul 8, 2015 at 1:22 PM, Dmitry Olshansky <notifications@github.com>
wrote:

> In simd.dd
> <https://github.com/D-Programming-Language/dlang.org/pull/1033#discussion_r34173809>
> :
>
> > @@ -92,8 +100,8 @@ $(H3 Conversions)
> >  	to their vector equivalents:)
> >
> >  ---
> > -int4 v = 7;
>
> What's the recommended way to multiply a float vector by a constant using
> SIMD then? I mean a compile ought to be able to replicate a vector of 3's
> and do the normal vector x vector SIMD multiply.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/D-Programming-Language/dlang.org/pull/1033/files#r34173809>
> .
>